lib/policies/rate-limit/rate-limit.js

```diff
@@ -1,4 +1,5 @@
 const RateLimit = require('express-rate-limit');
+// const SlowDown = require('express-slow-down');
```

Comment:
If we don't need this line, just remove it.

Reply:
It's work in progress, we'll definitely need this line. `express-slow-down` contains the deprecated functions.